test/functional/test_framework/blocktools.py

```diff
@@ -139,7 +140,8 @@ def create_coinbase(height, pubkey=None, extra_output_script=None, fees=0, nValu
     if pubkey is not None:
         coinbaseoutput.scriptPubKey = key_to_p2pk_script(pubkey)
     else:
-        coinbaseoutput.scriptPubKey = CScript([OP_TRUE])
+        # Use three OP_TRUEs to ensure we're always > 64 bytes in non-witness size
```

Comment:
it's not three `OP_TRUE`s?

Reply:
Find it hard to get reviews on your PR? Try our new patented icebreaker and conversation starter: Two OP_TRUEs and a lie!